src/pass.cpp

```
@@ -448,45 +448,67 @@ auto Pass::listKeys(const QString &keystring, bool secret) -> QList<UserInfo> {
  * @param err Raw stderr from GPG
  * @return Translated human-readable error, or empty string if not recognised
  */
+namespace {
+
+bool containsAny(const QString &str, const QStringList &patterns) {
```

Clang (14.0.6)

[error] use a trailing return type for this function

(modernize-use-trailing-return-type,-warnings-as-errors)